<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_file">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontal1">
     <item>
      <widget class="QLabel" name="label_section">
       <property name="text">
        <string>Sections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_section">
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_add_section">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/new/prefix1/img/icons8-plus-48.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_section">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/new/prefix1/img/icons8-cross-mark-48.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Additing</string>
       </property>
       <widget class="QPushButton" name="pushButton_add_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_type1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>32</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_type1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>186</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_key">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>186</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_value">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>186</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_value">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Deleting</string>
       </property>
       <widget class="QPushButton" name="pushButton_delete_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_key2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_type2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_file"/>
    <addaction name="separator"/>
    <addaction name="open_directory"/>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="save_file"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_file"/>
   <addaction name="separator"/>
   <addaction name="open_directory"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="save_file"/>
  </widget>
  <action name="open_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/img/icons8-opened-folder-50.png</normaloff>:/new/prefix1/img/icons8-opened-folder-50.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="open_directory">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/img/icons8-active-directory-50.png</normaloff>:/new/prefix1/img/icons8-active-directory-50.png</iconset>
   </property>
   <property name="text">
    <string>Открыть директорию</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/img/save2.png</normaloff>:/new/prefix1/img/save2.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/img/icons8-add-file-30.png</normaloff>:/new/prefix1/img/icons8-add-file-30.png</iconset>
   </property>
   <property name="text">
    <string>Создать новый файл</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
